lib/UI/workout/workout_page.dart

```diff
@@ -267,11 +262,7 @@ class _WorkoutPageState extends State<WorkoutPage> {
       onPressed: () {
         saveWorkout(workoutData.workout);
         Navigator.of(context).pop();
-        Navigator.of(context).push(
-          MaterialPageRoute(
-            builder: (context) => CalendarScreen(),
-          ),
-        );
+        Navigator.of(context).pop();
```

Comment:
Same as a previous comment, you can use .. when calling a method on the same reference twice.